core/src/main/java/hudson/model/AbstractBuild.java

```diff
@@ -338,7 +338,7 @@ public boolean shouldCalculateCulprits() {
     @Override
     @Nonnull
     public Set<User> calculateCulprits() {
-        Set<User> c = RunWithSCM.super.calculateCulprits();
+        Set<User> users = RunWithSCM.super.calculateCulprits();
```

Comment:
This more readable version instead ```c``` variable :spaghetti:

Reply:
Totally unrelated to the PR tho. Better to submit in separate PRs